codesign/lib/src/google_cloud_storage.dart

```diff
@@ -75,7 +64,7 @@ class EngineArtifactTransfer {
   }
 
   /// Utility function to download a file from google cloud.
-  Future<File> defaultDownloadFunction({
+  Future<File> downloadFunction({
     required String sourceUrl,
     required String localPath,
     required ProcessManager processManager,
```

Comment:
Instead of passing ProcessManager and FileSystem on every function call, it can be passed when `GoogleCloudStorage` is initialized

Reply:
Good advice! I refactored this class